assets/js/modules/tagmanager/components/setup/SetupForm.js

```diff
@@ -181,6 +182,8 @@ export default function SetupForm( { finishSetup } ) {
 				<TagCheckProgress />
 			</div>
 
+			<SetupUseSnippetSwitch />
```

Comment:
Per the ACs, in the setup this toggle should only show _if_ there is an existing Tag Manager tag. So you need to wrap this in a corresponding condition.

Also, this is a small detail that wasn't mentioned in the ACs, but could you move this below `ContainerNames`? When the field to enter a container name is shown, it makes more sense to have that above the snippet toggle.